src/test/regress/sql/join_hash.sql

```diff
@@ -593,16 +593,19 @@ ROLLBACK;
 -- re-use of the inner hash table across rescans.
 begin;
 set local enable_hashjoin = on;
+-- GPDB: Join with the replicated table to avoid passing the parameter through motion.
+create table int4_tbl_repl (like int4_tbl) distributed replicated;
+insert into int4_tbl_repl select * from int4_tbl;
 
 explain (costs off)
 select i8.q2, ss.* from
 int8_tbl i8,
-lateral (select t1.fivethous, i4.f1 from tenk1 t1 join int4_tbl i4
+lateral (select t1.fivethous, i4.f1 from tenk1 t1 join int4_tbl_repl i4
          on t1.fivethous = i4.f1+i8.q2 order by 1,2) ss;
 
 select i8.q2, ss.* from
 int8_tbl i8,
-lateral (select t1.fivethous, i4.f1 from tenk1 t1 join int4_tbl i4
+lateral (select t1.fivethous, i4.f1 from tenk1 t1 join int4_tbl_repl i4
```

Comment:
I think it is more logical to do it this way:
```suggestion
lateral (select t1.fivethous, i4.f1 from tenk1_repl t1 join int4_tbl i4
```